### ISSUE TITLE
Empty playlists/albums

### ISSUE THREAD

**Issue by tejas-git64** [sumitkolhe/jiosaavn-api#90]
The albums/playlists fetched by genres, but most of them have no songs/tracks when opening them. This problem started appearing ever since I hosted this as my own instance on vercel.

**Comment by sumitkolhe** [comment]
Did you host it in Mumbai region on vercel?

**Comment by tejas-git64** [comment]
> Did you host it in Mumbai region on vercel? 

Yep, i rehosted using your latest code, so now it kinda works?

**Comment by sumitkolhe** [comment]
Yeah it should work if the serverless function region is mumbai. Jiosaavn Serves songs based on the geo IP

**Comment by tejas-git64** [comment]
> Yeah it should work if the serverless function region is mumbai. Jiosaavn Serves songs based on the geo IP

Although some playlists/albums turn out to be empty at times, any issues with that?
You can check it out here: https://lowkeymusic-v2.netlify.app

**Comment by tejas-git64** [comment]
> Yeah it should work if the serverless function region is mumbai. Jiosaavn Serves songs based on the geo IP

Also is the homedata request removed permanently?

**Comment by actovisionitsol** [comment]
Playlists only have 10 songs, can it be more like the original jiosaavn one?, also, please add radio of an artist API endpoint.
